tools/profile.py

```diff
@@ -138,6 +140,7 @@ def stack_id_err(stack_id):
 // This code gets a bit complex. Probably not suitable for casual hacking.
 
 int do_perf_event(struct bpf_perf_event_data *ctx) {
+    CPU_FILTER
```

Comment:
You don't need to do it in the BPF program. Just call `attach_perf_event` with specific CPU.